engines/saga/itedata.cpp

```diff
@@ -590,6 +643,12 @@ const char *optionsStr[][4] = {
 		"\"Si, grazie. Ne avrei bisogno.\"",
 		"\"No, grazie, voglio provare a risolverlo da solo.\"",
 		"Penso che la tessera %s sia nel posto sbagliato."
+	},
+	{
+		"\"Je r\202soudrai cette \202nigme plus tard.\"",
+		"\"Oui, j'aimerais un indice s'il vous plait.\"",
+		"\"Non merci, je voudrais r\202soudre cela par moi même.\"",
```

Comment:
Should that be 'm\210me' rather than 'même'?

Reply:
yes same, i forgot to replace the two ê :(